src/components/table-of-contents/example-table-of-contents-sticky-full-page.njk

```diff
@@ -0,0 +1,114 @@
+---
+title: "Example: Table of contents on full page"
+layout: ~
+---
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+{% extends "layout/_template.njk" %}
+
+{% set pageConfig = {
+    "header": {
+        "title": "What is the census?"
```

Comment:
Might need to check with Oscar but I think census should have a upper case C

Reply:
Cool cheers. Yeah I noticed that, some upper case and some lower case. I feel like it could be something that might need an upper case C but I could be wrong

Reply:
Haven't heard back yet, will spin this off into another ticket